### ISSUE TITLE
Can't copy to clipboard

### ISSUE THREAD

**Issue by PLatCIC** [SoarGroup/VisualSoar#11]
It’s almost impossible to copy text from the editor or the main window and get it into the clipboard so it can be pasted into some other document.

**Comment by johnlaird** [comment]
Could not reproduce.
I just copied:
sp {apply*erase-region
   (state <s> ^operator <op>
              ^old-region <region>
              ^io.output-link.region.status complete)
   (<op> ^name erase-region
         ^region <region>)
-->
   (<s> ^old-region <region> -)
}
by selecting with my mouse (left-click and hold) and ctrl c.  Not sure how this could be a Visual-Soar issue as opposed to some other part of the software pipeline.

**Comment by PLatCIC** [comment]
I could not reproduce it today either. Yesterday I was trying to copy operator names, one at time, from the left-hand VS pane and paste them into a Word doc. It worked the first time I tried it, but failed after that. I would copy something and then try to paste it and I would always get the original thing I had copied successfully instead of the new thing I had just tried to copy. Today I tried the same thing and it worked just fine. I'll close this for now, but if it happens again I'll try to capture more details.
